src/restic/node.go

```diff
@@ -496,9 +568,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 		err = errors.Errorf("invalid node type %q", node.Type)
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
```

Comment:
Same as above, you don't need this switch. `fillAcls()` will call functions that are no-ops on unsupported OS.